pkgs/tools/text/par/default.nix

```
@@ -18,7 +22,7 @@ stdenv.mkDerivation {
     })
   ];
 
-  buildPhase = ''make -f protoMakefile'';
+  buildPhase = ''make -f protoMakefile CC="${cc} -c" LINK1=${cc}'';
```

targetPrefix should be empty for non-cross builds.

```suggestion
  buildPhase = ''make -f protoMakefile CC="${stdenv.cc.targetPrefix}cc -c" LINK1=${stdenv.cc.targetPrefix}cc'';
```